matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/handler/PresenceSyncHandler.kt

```diff
@@ -30,24 +31,26 @@ import javax.inject.Inject
 internal class PresenceSyncHandler @Inject constructor() {
 
     fun handle(realm: Realm, presenceSyncResponse: PresenceSyncResponse?) {
-        presenceSyncResponse?.events
-                ?.filter { event -> event.type == EventType.PRESENCE }
-                ?.forEach { event ->
-                    val content = event.getPresenceContent() ?: return@forEach
-                    val userId = event.senderId ?: return@forEach
-                    val userPresenceEntity = UserPresenceEntity(
-                            userId = userId,
-                            lastActiveAgo = content.lastActiveAgo,
-                            statusMessage = content.statusMessage,
-                            isCurrentlyActive = content.isCurrentlyActive,
-                            avatarUrl = content.avatarUrl,
-                            displayName = content.displayName
-                    ).also {
-                        it.presence = content.presence
-                    }
+        if (BuildConfig.PRESENCE_SYNC_ENABLED) {
```

Comment:
for the SDK we should avoid the interacting directly with the build config and use https://github.com/vector-im/element-android/blob/develop/matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/MatrixConfiguration.kt

then the creation of the config can use the vector `BuildConfig` set by the application https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/core/di/SingletonModule.kt#L117